.env.example

```diff
@@ -491,3 +491,7 @@ SALESFORCE_GRAPHQL_MAX_RETRIES=3
 
 # Force Node.js to use UTC as process timezone
 TZ=UTC
+
+# Dummy oauth client for platform testing (optional)
```

Comment:
It is for the examples app "packages/platform/examples/base" and indeed future testing